jobs/views.py

```diff
@@ -1,5 +1,24 @@
-from django.shortcuts import render
+from django.shortcuts import render, get_object_or_404
+from django.template.response import TemplateResponse
+
+from .models import Job
 
```

Comment:
these views could be changed to generic class views (and use the queryset argument or get_queryset() method) to simplify the code...but this can wait till a later task